build/html/docs/api_description.md

```diff
@@ -0,0 +1,136 @@
+| 3      | Machine Size         | Compute Instance Type         |
+
+
+# Authentication
+
+## Bearer Auth
+
+The HPE GreenLake for bare metal APIs uses Bearer Authentication that requires the users to provide 
+a bearer token in the **Authorization** header.   The service supports two types of tokens, the HPE 
+GreenLake IAM token and the Metal classic token.  The Metal classic token is HPE internal purpose only.
+The clients are required to provide HPE GreenLake IAM token to authenticate with the HPE GreenLake 
+for bare metal APIs.  The access token can be obtained using any of the following way.
+
+<h2>Option1: API client(recommended)</h2>
+
+An API client allows nonhuman entities (an application service account, for instance) programmatic access to a resource on a space.
+
+  + 1.
+<a
+    href="https://support.hpe.com/hpesc/public/docDisplay?docId=a00092451en_us&page=greenlakecentral-create-api-client.html"
+    target="_blank">
+    Create an API client
+    <img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/share.svg"
+    height="12">
+</a>
+
+**Note:** Make sure to save the **Issuer Url**, **Client ID**, and **Client Secret**
+
+  + 2.
+<a
+    href="https://support.hpe.com/hpesc/public/docDisplay?docId=a00092451en_us&page=GUID-1CEA233B-C4B0-41B7-9A25-7A36D9FC0312.html"
+    target="_blank">
+    Creating an assignment for an API client
+    <img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/share.svg"
+    height="12">
+</a>
+
+**Note:** Make sure to assign the appropriate  HPE GreenLake for bare metal user roles in HPE GreenLake Central.
+
+  + 3. Make a REST call to generate the API access token:
+
+```
+curl -i -X POST \
+  '{issuerUrl}/v1/token' \
+  -H 'Content-Type: application/x-www-form-urlencoded' \
+  -d 'client_id={clientID}' \
+  -d 'client_secret={clientSecret}' \
+  -d grant_type=client_credentials \
+  -d scope=hpe-tenant
+```
+Obtain the `access_token` from the response.
+
+<h2>Option2: Getting the token directly from UI</h2>
+
+To authenticate with the BMaaS API you need to obtain the access token from
+<a
+    href="https://client.greenlake.hpe.com"
+    target="_blank">
+    HPE GreenLake Central
+    <img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/share.svg"
+    height="12">
+</a>.
+Once logged in to **HPE GreenLake Central**,
+Click the
+<img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/user.svg">
+icon in the top right corner and then click
+<input
+    type="button"
+    value="API Access"
+    style="color: #000;
+        background-color: #fff;
+        border: 2px solid #00b388;
+        border-radius: 4px;"/>
+. In the API Access page, click the
+<img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/clipboard.svg">
+icon to copy the personal access token.
+
+<br>
+
+> <img src="https://raw.githubusercontent.com/grommet/grommet-icons/stable/img/circle-information.svg"> **INFO**
```

Comment:
Is there a mismatch of bracket here?  The leading ">" doesn't seem to have a match.

Reply:
that was a typo. removed it.